src/runtime/module_jit_ref_count.cpp

```diff
@@ -0,0 +1,29 @@
+#include "./runtime_internal.h"
+
+extern "C" {
+
+WEAK void *halide_jit_module_argument = NULL;
+WEAK void (*halide_jit_module_adjust_ref_count)(void *arg, int32_t count) = NULL;
+
+}
+
+namespace Halide { namespace Runtime { namespace Internal {
+
+WEAK void halide_use_jit_module() {
```

Comment:
These appear to be a way for the runtime itself to tell libHalide not to free the jit-compiled module (e.g. because there's an outstanding device buffer). A comment explaining that would be good.